### PR TITLE
feat(GlobalGiving): 19969 new Layer Features panel items for GlobalGiving

### PR DESCRIPTION
**Pull request by albaranau**
https://kontur.fibery.io/Tasks/Task/FE-Layer-Features-panel-for-GlobalGiving-19969

To test locally:
- run localhost on TEST stage
- open https://localhost:3000/map?app=52b9efd2-0527-4236-9bb6-9677bea1d790
- make sure you have a selected area that intersects with some of the target countries (e.g. Iraq, Myanmar, Ukraine)

image

## Release Notes

- **New Features**
  - Introduced functionality for generating feature cards based on ACAPS datasets.
  - Added a new `CardText` component for enhanced text display in feature cards.
  - Enhanced `LayerFeaturesPanel` to support dynamic headers and localized text.

- **Improvements**
  - Improved feature loading mechanism with new layer IDs for better flexibility.
  - Updated the `FullState` component to conditionally display additional information.
  - Enhanced error handling for feature focus in the `LayerFeaturesPanel`.
  - Refined conditional checks for subscription features to prevent runtime errors.

- **Style**
  - Added new CSS styles for text and title elements to standardize typography.

- **Type Definitions**
  - Expanded TypeScript interfaces for ACAPS features to improve data structure representation.
  - Introduced a new interface for `LayerFeaturesPanelConfig` to enhance configuration management.

### CODE REVIEW

File: src/features/layer_features_panel/components/LayerFeaturesPanel/index.tsx [hunk 1]
```diff
@@ -84,7 +99,11 @@ export function LayerFeaturesPanel() {
   return (
     <>
       <Panel
-        header={FEATURESPANEL_HEADER}
+        header={
+          featuresPanelLayerId === HOT_PROJECTS_LAYER_ID
+            ? HOT_PROJECTS_HEADER
+            : ACAPS_DATA_HEADER
+        }
```
[the remark this answers is not in the document]
Comment (albaranau) [reply src/features/layer_features_panel/components/LayerFeaturesPanel/index.tsx:106]: @coderabbitai Where is GLOBALGIVING_LAYER_ID defined?

[the remark this answers is not in the document]
Comment (albaranau) [reply src/features/layer_features_panel/components/LayerFeaturesPanel/index.tsx:106]: @coderabbitai Why did you decide that we need `GLOBALGIVING_LAYER_ID`?

[the remark this answers is not in the document]
Comment (albaranau) [reply src/features/layer_features_panel/components/LayerFeaturesPanel/index.tsx:106]: @coderabbitai there's no GlobalGiving layer, you made it up

File: src/core/shared_state/referenceArea.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ function getReferenceAreaFromConfigRepo(): GeometryWithHash | null {
   const features = configRepo.get().features;
   // if there's a geometry in reference_area configuration - use it for initialization
   const refAreaGeometry =
+    features[FeatureFlag.REFERENCE_AREA] &&
```
Comment (albaranau) [inline src/core/shared_state/referenceArea.ts:18]: Additional null check, not related to this task

File: src/views/Pricing/Pricing.tsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ import type { SubscriptionsConfig } from '~features/subscriptions/types';
 export function PricingPage() {
   const subscriptionFeature = configRepo.get().features[FeatureFlag.SUBSCRIPTION];
   const config =
-    typeof subscriptionFeature === 'object'
+    subscriptionFeature && typeof subscriptionFeature === 'object'
```
Comment (albaranau) [inline src/views/Pricing/Pricing.tsx:12]: Additional null check, not related to this task

**Review by vkozio — APPROVED** [review]

**Pull request merged**